src/python/tests/core/bot/tasks/fuzz_task_test.py

```diff
@@ -1368,7 +1369,7 @@ def test_basic(self):
     self.assertEqual(1, crashes[0].return_code)
     self.assertEqual('stack', crashes[0].unsymbolized_crash_stacktrace)
     self.assertEqual(1.0, crashes[0].crash_time)
-    self.assertListEqual(['test_target', 'args'], crashes[0].arguments)
+    self.assertEqual('args', crashes[0].arguments)
```

Comment:
shouldn't this be a list?

Reply:
This was wrong before. This needs to be same type as Testcase.minimized_arguments, which is a string.